mern/server/controller/flow.js

```diff
@@ -1,83 +1,24 @@
-import Flow from "../models/flow.model.js"
-import parse from "../utils/parse.js";
+import FlowInteractor from "../interactor/flow_interactor.js";
 import asyncHandler from "express-async-handler";
 
-export const getFlow = asyncHandler(async(req, res) => {
+//Controllers for REST APIs for the Flow entity
 
```

Comment:
You need to create an instance of the FlowInteractor class. 

let flowinteractor = new FlowInteractor(). This will then allow you to use it in your arrow functions